StaticAnalyzer/views/android/strings.py

```diff
@@ -12,30 +13,27 @@
 )
 
 
-def strings_jar(app_file, app_dir, tools_dir):
+def strings(app_file, app_dir, tools_dir):

```

Comment:
Rename this to `strings_jar`
else this will break the code
https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/python3/StaticAnalyzer/views/android/static_analyzer.py#L43

Reply:
remove `tools_dir` from here as well as from the caller. It's not required.